### ISSUE TITLE
Dedicated Docker image?

### ISSUE THREAD

**Issue by bartmans** [dysproseum/kplaylist-php7#7]
Hi,

Are you still planning on providing a dedicated Docker image voor Kplaylist? I for one could use one :-)

**Comment by dysproseum** [comment]
Yes the docker image is still planned, it will likely be a docker-compose file that includes versions of php, apache, and mysql.

Thanks for the interest :-)

**Comment by bartmans** [comment]
I'm not a developer but can test for you. Am using Airsonic-advanced now but sometimes simple is better and kplaylist does exactly what I want.

**Comment by dysproseum** [comment]
Thanks, I put together a new repository that hosts the docker configuration, it will pull this one into its www docroot as a submodule:

https://github.com/dysproseum/kplaylist-docker

Let me know if this works when you can test this out.
